src/servers/src/http/authorize.rs

```diff
@@ -68,7 +66,7 @@ pub async fn inner_auth<B>(
         .current_schema(schema.clone())
         .timezone(timezone);
 
-    let query_ctx = Arc::new(query_ctx_builder.build());
+    let query_ctx = query_ctx_builder.build();
```

Comment:
Should we add `Channel::Http` as build default value here?

Reply:
Because http as a transport channel is shared by a few protocols, so we will specify the channel in each particular handler